libs/tokens/src/utils/getTokenLogoUrls.ts

```diff
@@ -23,3 +18,12 @@ export function getTokenLogoUrls(token: TokenWithLogo | undefined): string[] {
 
   return urls
 }
+
+function getTokenLogoFallbacks(address: string, chainId: SupportedChainId): string[] {
+  return [
+    cowprotocolTokenUrl(address, chainId),
+    cowprotocolTokenUrl(address.toLowerCase(), chainId),
```

Comment:
Now lowercase is the default, right?
Should it then come first, maybe?

Reply:
Yes, now the first fallback is irrelevant. Thanks!